<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Robot Console</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>820</width>
     <height>560</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>820</width>
     <height>560</height>
    </size>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>610</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QWidget" name="robotControlWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>58</y>
          <width>62</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Engine 1</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>98</y>
          <width>62</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Engine 2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>138</y>
          <width>62</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Engine 3</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>178</y>
          <width>62</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Engine 4</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>218</y>
          <width>62</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Engine 5</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>258</y>
          <width>62</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Engine 6</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>212</x>
          <y>30</y>
          <width>61</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Movement</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="btnEngine1Forward">
        <property name="geometry">
         <rect>
          <x>214</x>
          <y>58</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnEngine1Backward">
        <property name="geometry">
         <rect>
          <x>244</x>
          <y>58</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnEngine2Backward">
        <property name="geometry">
         <rect>
          <x>244</x>
          <y>98</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnEngine2Forward">
        <property name="geometry">
         <rect>
          <x>214</x>
          <y>98</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnEngine3Forward">
        <property name="geometry">
         <rect>
          <x>214</x>
          <y>138</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnEngine3Backward">
        <property name="geometry">
         <rect>
          <x>244</x>
          <y>138</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnEngine4Forward">
        <property name="geometry">
         <rect>
          <x>214</x>
          <y>178</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnEngine4Backward">
        <property name="geometry">
         <rect>
          <x>244</x>
          <y>178</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnEngine5Forward">
        <property name="geometry">
         <rect>
          <x>214</x>
          <y>218</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnEngine5Backward">
        <property name="geometry">
         <rect>
          <x>244</x>
          <y>218</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnEngine6Close">
        <property name="geometry">
         <rect>
          <x>244</x>
          <y>258</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnEngine6Open">
        <property name="geometry">
         <rect>
          <x>214</x>
          <y>258</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="txtReceive">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>327</y>
          <width>241</width>
          <height>201</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>300</y>
          <width>61</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Log</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtEngine1Receiving">
        <property name="geometry">
         <rect>
          <x>87</x>
          <y>59</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtEngine2Receiving">
        <property name="geometry">
         <rect>
          <x>87</x>
          <y>99</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtEngine3Receiving">
        <property name="geometry">
         <rect>
          <x>87</x>
          <y>139</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtEngine4Receiving">
        <property name="geometry">
         <rect>
          <x>87</x>
          <y>179</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtEngine5Receiving">
        <property name="geometry">
         <rect>
          <x>87</x>
          <y>219</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtEngine6Receiving">
        <property name="geometry">
         <rect>
          <x>87</x>
          <y>259</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="drawBoxLayout" stretch="0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnRotateRight">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/res/Images/rotate_right.png</normaloff>:/res/Images/rotate_right.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRotateLeft">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/res/Images/rotate_left.png</normaloff>:/res/Images/rotate_left.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnZoomout">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/res/Images/zoom_out.png</normaloff>:/res/Images/zoom_out.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnZoomin">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/res/Images/zoom_in.png</normaloff>:/res/Images/zoom_in.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSerial_Port"/>
    <addaction name="actionServer"/>
    <addaction name="actionClient_2"/>
   </widget>
   <widget class="QMenu" name="menuCommunication">
    <property name="title">
     <string>Communication</string>
    </property>
    <addaction name="actionTCPServer"/>
    <addaction name="actionClient"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuCommunication"/>
  </widget>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
   <property name="toolTip">
    <string>File</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSerial_Port">
   <property name="text">
    <string>Serial Port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionServer">
   <property name="text">
    <string>Server</string>
   </property>
  </action>
  <action name="actionTCPServer">
   <property name="text">
    <string>Server</string>
   </property>
   <property name="toolTip">
    <string>Server</string>
   </property>
  </action>
  <action name="actionClient">
   <property name="text">
    <string>Client</string>
   </property>
   <property name="toolTip">
    <string>Client</string>
   </property>
  </action>
  <action name="actionClient_2">
   <property name="text">
    <string>Client</string>
   </property>
   <property name="toolTip">
    <string>Client</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
    Æ¿Ê_Ê˜a∑ç˜ËŒ_m≥sA^EÑËMO=›vdÿﬁÉñj™—êLıö[‡Æ‡ Ã¯Ü–j-‹Ω{«–<!–«`ËÇˆÍó±ßˆBkòûJ”Fˆ:D‘$ª?Lª4±ºf¸˜`oáû®ˆ$©‚˘Ñ™Éñ^&^ïw€ƒˆÍ”+yÊBäpÏôzB{1ô90ïîõwˇ6˝ò¡úÑ·UZ.u+ ûtﬁ33Ãvåg"y¨L∆Œ∞ªWúøDë%